docs/product/components/tables.html

```diff
@@ -29,9 +28,59 @@
     </tbody>
 </table>
 {% endhighlight %}
-
         <div class="stacks-preview--example">
             <table class="s-table">
+                <thead>
+                    <tr>
+                        <th scope="col">Name</th>
+                        <th scope="col">Reputation</th>
+                        <th scope="col">Joined</th>
+                        <th scope="col">Last Seen</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <th><a>Aaron Shekey</a></th>
```

Comment:
`tbody > tr > th` – is this correct use of `th`? I thought it can be used only in `tbody`..? I'm not sure to be honest, just thought I'll point it out. Feel free to ignore if I'm wrong.

Reply:
`th`s are used for any header type cell regardless of `thead`, `tbody`, or `tfoot`. ([Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th))

Reply:
Oh, I didn't know that, thanks!!